src/app/features/home/search-bar/search-bar.service.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpClient } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+import { API_URL } from '@core/env.token';
+import { combineLatest, map } from 'rxjs';
+import { Event, Group } from './search-bar.interface';
+
+@Injectable()
+export class SearchBarService {
+  private API_URL = inject(API_URL);
+  private http = inject(HttpClient);
+
+  search(value: string) {
+    // This whole combineLatest will be replaced with one endpoint.
+    return combineLatest([
+      this.http.get<Event[]>(`${this.API_URL}events?name_like=${value}`),
+      this.http.get<Group[]>(`${this.API_URL}groups?name_like=${value}`),
```

Comment:
`${this.API_URL}events?name_like=${value}` mysle ze mozemy ogarnac sobie jakas prywatna metode w serwisie ktora przyjmie endpoint i wartosc `value` i zwroci nam gotowy url

Reply:
Tutaj finalnie będzie jeden endpoint:  `api/v1/search?value=""`. Teraz są dwa endpointy, bo cięzko to zmockować w json serverze w ten sposób, dlatego nie wiem czy jest sens to robić